### ISSUE TITLE
for now ,We need add some demo for downloaders to try this good programme

### ISSUE THREAD

**Issue by woodgreat** [CodeCrafter-TL/tkinter-macOS#2]
for now ,We need add some demo for downloaders to try this good programme

**Comment by TotoWang-hhh** [comment]
but seems the author is still developing some basic things **for** the apps.

**Comment by TotoWang-hhh** [comment]
> for now ,We need add some demo for downloaders to try this good programme

let me tell you a secret. this program just sets background images for your tkinter components to make it looks like mac os ui, and you can't even set the size of the button for now!

So what I really wanna tell you is that this program still has a long way from building some demos.

**Comment by Mengobs** [comment]
Yes, as Totowang-hhh said, our project is not perfect at present and needs a long time to develop.

**Comment by woodgreat** [comment]
I count on your work ! thank you
